set-integration-testing/src/integration-test/java/eu/europeana/api/set/integration/web/WebUserSetPublishingIT.java

```diff
@@ -70,12 +72,13 @@ public void publishUnpublishUserSet_Success() throws Exception {
 
     // publish set by publisher
     // expected change of ownership to editorial team
-    publishUserSet(userSet, getConfiguration().getEuropeanaPublisherNickname());
+    String issued = "2018-10-31T01:30:00.000Z";
+    publishUserSet(userSet, issued, getConfiguration().getEuropeanaPublisherNickname());
```

Comment:
response must be validated

Reply:
It is validated in the method publishUserSet.